src/pages/IndividualArticle.jsx

```diff
@@ -17,7 +21,25 @@ const IndividualArticle = () => {
         }).catch((err) => {
             setError({ err });
         })
-    }, [article_id, article])
+    }, [article_id])
+
+    const handleVote = (vote) => {
+        if(hasVoted) alert('You have already voted on this article');
```

Comment:
From a UX perspective, do we need an alert every time we try to vote again? I think the button not working again is enough of an indicator.

Also, what if I wanted to retract my vote by using the opposite button? Currently, I only have one vote(which is good) but it can't be undone. I might get further down an article and change my mind.